contributors.md

```diff
@@ -41,8 +42,11 @@ I am a Mozilla fellowship (2016) and Fulbright scholarship (2007) alumna.
 
 **Contributors names should be added alphabetically**
 
+<!---A--->
+
 ### Alexander Morley
 
+* Role: Previous Infratructure Maintainer
```

Comment:
```suggestion
* Role: Infrastructure Maintainer (2018 - 2019)
```

Rather than saying "previous" can we put the dates that someone did the role? 

That might scale well to having people change roles? Or maybe we could have `Role` be the current role and then an optional bullet point for `Previous Role(s)`?